frontend/public/components/chargeback.tsx

```diff
@@ -317,23 +233,21 @@ class ReportData extends React.Component<ReportDataProps, ReportDataState> {
 
   render() {
     const {obj} = this.props;
-    const phase = _.get(obj, ['status', 'phase']);
-
-    const {data, reduceBy, sortBy, orderBy, keys, rows, maxValues, totals, inFlight, error} = this.state;
+    const conditions = _.get(obj, 'status.conditions');
+    const isFinished = _.some(conditions, {type: 'Running', status: 'False'});
+    const {data, sortBy, orderBy, keys, rows, inFlight, error} = this.state;
 
     let dataElem = <MsgBox title="No Data" detail="Report not finished running." />;
     if (inFlight) {
       dataElem = <div className="row"><div className="col-xs-12 text-center"><LoadingInline /></div></div>;
     } else if (error) {
-      dataElem = <LoadError label="Report" message={error} />;
-    } else if (phase === 'Finished') {
-      if (data) {
-        dataElem = <DataTable sortBy={sortBy} orderBy={orderBy} keys={keys} rows={rows} maxValues={maxValues} totals={totals} applySort={(sb, func, ob) => this.applySort(sb, func, ob)} />;
+      dataElem = <LoadError label="Report" message={_.get(error, 'json.error') || error.message} />;
+    } else if (isFinished) {
+      if (data && !data.error) {
```

Comment:
In the cases where `data.error` is set, we should show the error

Reply:
Fixed